src/bp-integrations/learndash/library/SyncGenerator.php

```diff
@@ -753,4 +753,61 @@ protected function getLastSyncTimestamp( $type = 'bp' ) {
 
 		return $lastSync;
 	}
+
+	/**
+	 * Update a ld group based on current bp group
+	 *
+	 * @param string $ld_group_id Group id for learndash
+	 *
+	 * @param int    $groupId Group id for buddyboss
+	 *
+	 * @since BuddyBoss 1.5.8
```

Comment:
@jitendrabanjara1991 Next release is `1.5.7`
```suggestion
	 * @since BuddyBoss 1.5.7
```